judging/scripts/prescribe/generate_prescriptions.sh

```diff
@@ -42,6 +42,10 @@ timeout_script="$repo_dir"/judging/scripts/timeout_killer.sh
 chmod +x "$timeout_script"
 $timeout_script &
 
+# Clean up timeout monitor process on exit
+timeout_script_pid=$!
+trap 'pkill -KILL --parent $timeout_script_pid' EXIT
```

Comment:
This means "if the script exits for any reason, run the stuff in quotes". I don't know if that includes `SIGKILL` (`kill -9`) as that signal isn't even delivered to the running process so I don't believe it can be trapped, by design.